packages/fx-core/src/component/error/componentError.ts

```diff
@@ -114,6 +114,33 @@ export class ExternalApiCallError extends BaseComponentInnerError {
     super("UserError", name, message, messageParams, suggestionKey, detail, helpLink);
     this.statusCode = statusCode;
   }
+
+  static getAzureCredentialError(error?: unknown): ExternalApiCallError {
+    error = error ?? "";
+    return new ExternalApiCallError(
+      "GetAzureCredentialError",
+      "plugins.bot.FailRetrieveAzureCredentials",
+      -1,
```

Comment:
Just curious that what does it  mean when value is -1?

Reply:
Nothing, I'll ignore the status code value if the value is -1 when I writing error log